docs/whatsnew/4.0.rst

```diff
@@ -25,17 +25,18 @@ We have bumped the minimum version of several packages we depend on; these are t
 
 Reworked internals of TimeSeries
 ================================
-`~sunpy.timeseries.GenericTimeseries` has been re-worked to use astropy time
+``sunpy.timeseries.GenericTimeseries`` has been re-worked to use astropy time
```

Comment:
This linking should be fine, it just needs `Timeseries` > `TimeSeries`. @nabobalis can you fix this? I don't have time at the moment.

Reply:
Sure.